config.go

```diff
@@ -152,17 +154,18 @@ func loadConfig() (*config, error) {
 		ReadMacPath:         defaultReadMacPath,
 		LogDir:              defaultLogDir,
 		PeerPort:            defaultPeerPort,
+		RPCHost:             defaultRPCHost,
```

Comment:
In the past we didn't enable allowing the RPC server to listen on an arbitrary interface as we didn't yet have TLS support as the default. Since then we've also added full support for macaroons as an authentication mechanism. As a result, we should _disallow_ having a _non-localhost_ target host while also having macaroons disabled.